vue.config.js

```diff
@@ -38,6 +41,10 @@ module.exports = {
     if (npm_package_version) {
       definitions['process.env.npm_package_version'] = JSON.stringify(npm_package_version);
     }
+    // eslint-disable-next-line camelcase
+    if (npm_package_dependencies__aeternity_aepp_sdk) {
+      definitions['process.env.SDK_VERSION'] = JSON.stringify(npm_package_dependencies__aeternity_aepp_sdk);
```

Comment:
I propose to preserve the original naming as we are doing above with `npm_package_version`